build.gradle

```diff
@@ -1243,7 +1243,7 @@ project(':clients') {
 
     testImplementation libs.bcpkix
     testImplementation libs.junitJupiter
-    testImplementation libs.mockitoCore
+    testImplementation libs.mockitoInline
```

Comment:
Could you explain why we need MockitoInline here? Any method only exists in `mockitoInline`?

Reply:
I needed this to mock out ChannelBuilders#createPrincipalBuilder and Sasl#createSaslServer. It's a petty there are these static calls, hard to test them.

Reply:
@acsaki , as @ijuma suggested, would you file another small PR to update the tests to avoid mocking the static methods? For the `ChannelBuilders#createPrincipalBuilder`, it looks like we actually don't need it for these tests, right? (the tests still pass after I removed them)

Reply:
bump, I second we should aim to remove this